### PR TITLE
Change IStudent

### PR DESCRIPTION
**Pull request by Bi1ku**

### CODE REVIEW

File: fs/index.ts [hunk 1]
```diff
@@ -4,6 +4,16 @@ export interface IStudent {
...
   password: string;
   email: string;
+  discordId?: string;
+  location?: string;
+  personalWebsite?: string;
+  hobbies?: string;
+  githubName?: string;
+  about?: string;
+  handle?: string;
+  dob?: string;
```
Comment (chuxu200328) [inline fs/index.ts]: date

File: fs/index.ts [hunk 2]
```diff
@@ -4,6 +4,16 @@ export interface IStudent {
   lastName: string;
   password: string;
   email: string;
+  discordId?: string;
+  location?: string;
+  personalWebsite?: string;
+  hobbies?: string;
+  githubName?: string;
+  about?: string;
+  handle?: string;
```
Comment (chuxu200328) [inline fs/index.ts]: what is handle?

File: fs/index.ts [hunk 3]
```diff
@@ -4,6 +4,16 @@ export interface IStudent {
...
+  dob?: string;
+  gender?: string;
```
Comment (chuxu200328) [inline fs/index.ts]: create a enum for gender

**Review by chuxu200328 — CHANGES_REQUESTED** [review]

File: fs/index.ts (hunk 2 above)
Comment (Bi1ku) [reply fs/index.ts]: username. I saw it like that in the template code of settings so I just changed username to handle

Comment (mxdi9i7) [reply fs/index.ts]: nah, use `username` instead.

**Review by mxdi9i7 — CHANGES_REQUESTED** [review]

**Comment by mxdi9i7** [comment]
and clean up your commits

File: fs/index.ts (hunk 2 above)
Comment (Bi1ku) [reply fs/index.ts]: alright

File: fs/index.ts [hunk 4]
```diff
@@ -11,10 +16,11 @@ export interface IStudent {
   githubName?: string;
   about?: string;
   handle?: string;
-  dob?: string;
-  gender?: string;
+  date?: string;
```
Comment (chuxu200328) [inline fs/index.ts]: lol, I mean the data type shoulde "Date". dob is fine.

File: fs/index.ts [hunk 5]
```diff
@@ -11,10 +16,11 @@ export interface IStudent {
...
+  gender?: gender;
```
Comment (chuxu200328) [inline fs/index.ts]: nume role for enum is similar to interface. This one should be EGender.

File: fs/index.ts [hunk 6]
```diff
@@ -11,10 +16,11 @@ export interface IStudent {
...
   avatar?: string;
   studentId: string;
+  likes: [];
```
Comment (chuxu200328) [inline fs/index.ts]: array of what

**Review by chuxu200328 — CHANGES_REQUESTED** [review]

**Review by chuxu200328 — APPROVED** [review]

**Pull request merged**